drivers/storage/portworx/component/telemetry.go

```diff
@@ -404,10 +408,43 @@ func (t *telemetry) deleteCCMGoPhonehomeCluster(
 	return nil
 }
 
+// PWX-27401 only reconcile metrics collector if it's already running, revert once collector memory issue got resolved
```

Comment:
```suggestion
// TODO
// PWX-27401 only reconcile metrics collector if it's already running, revert once collector memory issue got resolved
```

Reply:
I have a ticket tracking the revert, should be ok.